readthedocs/settings/base.py

```diff
@@ -228,15 +231,6 @@ def INSTALLED_APPS(self):  # noqa
     CELERY_CREATE_MISSING_QUEUES = True
 
     CELERY_DEFAULT_QUEUE = 'celery'
-    # Wildcards not supported: https://github.com/celery/celery/issues/150
-    CELERY_ROUTES = {
```

Comment:
I think this is not used anymore. None of those tasks exist.

Reply:
We've moved to marking the queue on `@task` declaration